docs/en/sql-reference/20-sql-functions/10-semi-structured-functions/json-array-insert.md

```diff
@@ -0,0 +1,33 @@
+---
+title: JSON_ARRAY_INSERT
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.644"/>
+
+Inserts a value into a JSON array at the specified index and returns the updated JSON array.
+
+## Syntax
+
+```sql
+JSON_ARRAY_INSERT(<json_array>, <index>, <json_value>)
+```
+
+| Parameter      | Description                                                 |
+|----------------|-------------------------------------------------------------|
+| `<json_array>` | The JSON array to modify.                                   |
+| `<index>`      | The position at which the value will be inserted (0-based). |
```

Comment:
`index` can also be a negative number, starting from the end of the array.
If `index` is less than 0, it is treated as the first position, and if `index` is greater than the length of the array, it is treated as the last position.